content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -193,6 +193,8 @@ JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.ty
 kubectl get pods -o json | jq '.items[].spec.containers[].env[]?.valueFrom.secretKeyRef.name' | grep -v null | sort | uniq
 
 # List all containerIDs of initContainer of all pods
+# Helpful when cleaning up stopped containers, while avoiding removal of initContainers.
+# Related to https://github.com/kubernetes/kubernetes/issues/86531
```

Comment:
I think it's OK for https://github.com/kubernetes/kubernetes/issues/86531 to link to this part of the cheat sheet, but the cheatsheet should not link to https://github.com/kubernetes/kubernetes/issues/86531

The idea of a cheat sheet is to be as compact as is feasible.

Reply:
sure thing, makes sense
removed reference to github issue